engine/language-server/src/main/scala/org/enso/languageserver/runtime/RuntimeFailureMapper.scala

```diff
@@ -37,6 +37,8 @@ final class RuntimeFailureMapper(contentRootManager: ContentRootManager) {
         ContextRegistryProtocol.InvalidStackItemError(contextId)
       case Api.ModuleNotFound(moduleName) =>
         ContextRegistryProtocol.ModuleNotFound(moduleName)
+      case Api.ModuleNotFoundForExpression(expressionId) =>
+        ContextRegistryProtocol.ModuleNotFoundForExpression(expressionId)
```

Comment:
This needs because we are doing the serialization/deserialization twice. Once between engine/runner and once between runner/language server. I'd love to see this simplified.